<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>306</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>257</width>
    <height>306</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>257</width>
    <height>306</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>配置</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>COM1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox11">
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBox12"/>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>COM2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox21"/>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="comboBox22"/>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>COM3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox31"/>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="comboBox32"/>
    </item>
    <item row="4" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>COM4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox41"/>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="comboBox42"/>
    </item>
    <item row="5" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>COM5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBox51"/>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="comboBox52"/>
    </item>
    <item row="6" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>COM6</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="comboBox61"/>
    </item>
    <item row="6" column="2">
     <widget class="QComboBox" name="comboBox62"/>
    </item>
    <item row="7" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>COM7</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="comboBox71"/>
    </item>
    <item row="7" column="2">
     <widget class="QComboBox" name="comboBox72"/>
    </item>
    <item row="8" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>COM8</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="comboBox81"/>
    </item>
    <item row="8" column="2">
     <widget class="QComboBox" name="comboBox82"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_old">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>默认值</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
